src/ILVerification/src/TypeVerifier.cs

```diff
@@ -61,21 +62,20 @@ public void VerifyInterfaces()
 
             // Look for duplicates and prepare distinct list of implemented interfaces to avoid 
             // subsequent error duplication
-            VirtualMethodAlgorithm virtualMethodAlg = _typeSystemContext.GetVirtualMethodAlgorithmForType(type);
             List<InterfaceMetadataObjects> implementedInterfaces = new List<InterfaceMetadataObjects>();
             foreach (InterfaceImplementationHandle interfaceHandle in interfaceHandles)
             {
                 InterfaceImplementation interfaceImplementation = _module.MetadataReader.GetInterfaceImplementation(interfaceHandle);
-                DefType interfaceType = _module.GetType(interfaceImplementation.Interface) as DefType;
-                if (interfaceType == null)
+                TypeDesc interfaceTypeDesc = _module.GetType(interfaceImplementation.Interface) as TypeDesc;
+                if (interfaceTypeDesc == null)
                 {
                     ThrowHelper.ThrowTypeLoadException(ExceptionStringID.ClassLoadBadFormat, type);
                 }
 
                 InterfaceMetadataObjects imo = new InterfaceMetadataObjects
                 {
-                    DefType = interfaceType,
-                    InterfaceImplementationHandle = interfaceHandle
+                    InterfaceTypeDesc = interfaceTypeDesc,
```

Comment:
Nit: I would call this just `InterfaceType`. We typically omit the `Desc` part on the variable names in CoreRT.